code/client/munkilib/fetch.py

```diff
@@ -133,7 +131,8 @@ def get_url(url, destinationpath,
 
     options = {'url': url,
                'file': tempdownloadpath,
-               'follow_redirects': follow_redirects,
+               'must_follow_redirects': follow_redirects,
+               'allow_redirects': munkicommon.pref('AllowHTTPRedirects'),
```

Comment:
I just don't like this complexity of two similarly-named redirect flags. Can't we just pass the right value (none, 'https', 'all') to getResourceIfChangedAtomically, and let it pass it along as well?